rasa/core/policies/flow_policy.py

```diff
@@ -633,6 +633,22 @@ def _find_earliest_updated_question(
                 return question_step
         return None
 
+    def should_flow_be_cancelled(self, tracker: DialogueStateTracker) -> bool:
+        """Test whether the current flow should be cancelled.
+
+        Args:
+            tracker: the conversation state tracker
+        Returns:
+        Whether the current flow should be cancelled
+        """
+        if (
```

Comment:
there is a helper on the tracker for this, `tracker.has_action_after_latest_user_message()`